playwright-e2e/config.ts

```diff
@@ -0,0 +1,42 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+const env = process.env.ENVIRONMENT || 'aat';
+
+// Data which can be reused across multiple tests. 
+// Called simply with "import config from '../config';" and then e.g. "config.caseworker.email" in a test.
+// Any new data added below will immediately be available wherever config is imported.
+const configuration = {
+  // URLs
+  idamUrl:
+    process.env.IDAM_API_URL || `https://idam-api.${env}.platform.hmcts.net`,
+  frontEndBaseURL:
```

Comment:
Can we call this something more specific like manageCaseBaseURL?
Same for the FE_BASE_URL environment variable.